pdf/io.go

```diff
@@ -0,0 +1,92 @@
+package pdf // import "github.com/Zenika/goru/pdf"
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+	"github.com/unidoc/unidoc/pdf"
+)
+
+func EnsureDocumentsDir() error {
+	if err := os.MkdirAll(getDocumentsPath(), 0755); err != nil {
+		return errors.Wrap(err, "Error while creating documents dir")
+	}
+
+	return nil
+}
+
+func GetDocumentPath(document string) string {
+	return filepath.Join(getDocumentsPath(), document+".pdf")
+}
+
+func getDocumentsPath() string {
+	return viper.GetString("server.documentsPath")
```

Comment:
Does Viper use cache ? Or maybe this could be done on init and store in a local variable